package.json

```diff
@@ -94,22 +92,21 @@
   "devDependencies": {
     "@angular-devkit/build-angular": "13.1.2",
     "@angular/cli": "13.1.2",
```

Comment:
The latest version for angular/cli is 13.1.3. Recentry, a few angular staffs have version inconsistency.
Could you check with `npx npm update`?